docs/apib/compliancePolicies.apib

```diff
@@ -0,0 +1,214 @@
+FORMAT: 1A
+
+# XClarity compliancePolicies
+
+# get details of all policies
+
+## GET /compliancePolicies
++ Response 200
+{
+   "policies":[{
+      "description":"DEV ONLY",
+      "id":"1427930734934",
+      "inUse":"true",
+      "filterType":"all",
+      "lastModified":"2015-04-01 16:25:34",
+      "name":"DEFAULT-2015-04-01",
+      "updateRule":"",
+      "userDefined":"no"
+   },
+   {"updateRule":"",
+      "description":"ddddd",
+      "id":"1428397839104",
+      "inUse":"false",
+      "filterType":"managed",
+      "lastModified":"This policy was created by USERID on 2015-04-07 05:10:39",
+      "name":"aaaa",
+      "updateRule":"",
+      "userDefined":"yes"
+   }]
+}
++ Response 400
++ Response 401
++ Response 404
++ Response 409
++ Response 500
+
+# retrieve the persisted compare result for servers to which a compliance policy is assigned.
+
+## GET /compliancePolicies/persistedResult
++ Response 200
+{
+   "all": [{
+      "chassis": "FBEF740B178F4EFAA846E7225EE256DC",
+      "cmms": [{
+         "deviceCompliant": ""."message": [],
```

Comment:
This line is broken the tests due to JSON format issue

Reply:
You can just remove the dot here and put the `"message"` prop on another line

Reply:
Thanks updated it